Framework/HistogramData/test/AddableTest.h

```diff
@@ -0,0 +1,92 @@
+#ifndef MANTID_HISTOGRAMDATA_ADDABLETEST_H_
+#define MANTID_HISTOGRAMDATA_ADDABLETEST_H_
+
+#include <cxxtest/TestSuite.h>
+
+#include "MantidHistogramData/Addable.h"
+#include "MantidHistogramData/FixedLengthVector.h"
+#include "MantidHistogramData/HistogramX.h"
+#include "MantidHistogramData/Iterable.h"
+#include "MantidHistogramData/VectorOf.h"
+
+using Mantid::HistogramData::detail::Addable;
+using Mantid::HistogramData::detail::FixedLengthVector;
+using Mantid::HistogramData::detail::Iterable;
+using Mantid::HistogramData::detail::VectorOf;
+using Mantid::HistogramData::HistogramX;
+
+class AddableTester : public VectorOf<AddableTester, HistogramX>,
```

Comment:
As a very minor point. I'm not a great fan of seeing that testing the Addable functionality requires brining in so many other mixins. Doesn't this feel more like an integration test rather than a test of `Addable`?